modules/breakpad/breakpad_linuxbsd_windows.cpp

```diff
@@ -0,0 +1,130 @@
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2021 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2021 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "breakpad.h"
+
+#include "core/io/dir_access.h"
+#include "core/os/os.h"
+
+#ifdef _WIN32
+#include <thirdparty/breakpad/src/client/windows/handler/exception_handler.h>
+#else
+#include <thirdparty/breakpad/src/client/linux/handler/exception_handler.h>
+#endif
+
+static google_breakpad::ExceptionHandler *breakpad_handler = nullptr;
+
+#ifdef _WIN32
+static bool dump_callback(const wchar_t* dump_path, const wchar_t* minidump_id, void* context,
+   EXCEPTION_POINTERS* exinfo, MDRawAssertionInfo* assertion, bool succeeded) {
+   wprintf(L"Crash dump created at: %s/%s.dmp\n", dump_path, minidump_id);
+   fwprintf(stderr, L"Crash dump created at: %s/%s.dmp\n", dump_path, minidump_id);
+#else
+static bool dump_callback(const google_breakpad::MinidumpDescriptor& descriptor, void* context, bool succeeded) {
+    printf("Crash dump created at: %s\n", descriptor.path());
+#endif
+    return succeeded;
+}
+
+void initialize_breakpad(bool register_handlers) {
+    if(breakpad_handler != nullptr) {
+        // TODO: error?
+        return;
+    }
+
+#ifdef _WIN32
```

Comment:
Probably want to use PLATFORM_WINDOWS, but it's not a big thing.

Reply:
PLATFORM_WINDOWS doesn't seem to be a thing? That's why I selected "_WIN32" as the thing to check for.

```
[hhyyrylainen@dakara godot]$ ag PLATFORM_WINDOWS
thirdparty/doctest/doctest.h
349:#define DOCTEST_PLATFORM_WINDOWS
2841:#ifdef DOCTEST_PLATFORM_WINDOWS
2859:#else // DOCTEST_PLATFORM_WINDOWS
2864:#endif // DOCTEST_PLATFORM_WINDOWS
2995:#if defined(DOCTEST_PLATFORM_WINDOWS)
2997:#else // DOCTEST_PLATFORM_WINDOWS
3000:#endif // DOCTEST_PLATFORM_WINDOWS
3007:#elif defined(DOCTEST_PLATFORM_WINDOWS)
3018:#else  // DOCTEST_PLATFORM_WINDOWS
3024:#endif // DOCTEST_PLATFORM_WINDOWS
3618:#ifdef DOCTEST_PLATFORM_WINDOWS
4186:#ifdef DOCTEST_PLATFORM_WINDOWS
4334:#else // DOCTEST_PLATFORM_WINDOWS
4413:#endif // DOCTEST_PLATFORM_WINDOWS
4421:#ifdef DOCTEST_PLATFORM_WINDOWS
5075:#ifdef DOCTEST_PLATFORM_WINDOWS
5078:#endif // DOCTEST_PLATFORM_WINDOWS
5284:#ifdef DOCTEST_PLATFORM_WINDOWS
5286:#else // DOCTEST_PLATFORM_WINDOWS
5288:#endif // DOCTEST_PLATFORM_WINDOWS
5373:#ifdef DOCTEST_PLATFORM_WINDOWS
5376:#endif // DOCTEST_PLATFORM_WINDOWS
5956:#ifdef DOCTEST_PLATFORM_WINDOWS
5990:#endif // DOCTEST_PLATFORM_WINDOWS
6342:#ifdef DOCTEST_PLATFORM_WINDOWS
6345:#endif // DOCTEST_PLATFORM_WINDOWS
```

versus:
```
[hhyyrylainen@dakara godot]$ ag _WIN32
core/math/static_raycaster.h
38:#if defined(_WIN32) && defined(_MSC_VER)

core/typedefs.h
75:#ifdef _WIN32

drivers/vulkan/SCsub
25:    env.AppendUnique(CPPDEFINES=["VK_USE_PLATFORM_WIN32_KHR"])

modules/gdnative/include/gdnative/gdnative.h
38:#if defined(_WIN32) || defined(__ANDROID__)
50:#else // !_WIN32 && !__APPLE__
58:#elif defined(_WIN32)

modules/raycast/godot_update_embree.py
239:#elif defined(_WIN32)

modules/breakpad/breakpad_linuxbsd_windows.cpp
36:#ifdef _WIN32
44:#ifdef _WIN32
62:#ifdef _WIN32
103:#ifdef _WIN32
116:#ifndef _WIN32
127:#ifdef _WIN32

platform/uwp/detect.py
161:    env.Append(CPPDEFINES=[("WINVER", winver), ("_WIN32_WINNT", winver), "WIN32"])

-- and there's a ton more matches after this...
```

Reply:
Edited: `WINDOWS_ENABLED`

Reply:
Yeah, I can change to that.

Reply:
Made that change.